pyanaconda/packaging/yumpayload.py

```diff
@@ -302,6 +302,12 @@ def _writeYumConfig(self, cache_dir=_yum_cache_dir):
         if self.data.packages.multiLib:
             buf += "multilib_policy=all\n"
 
+        if self.data.packages.timeout is not None:
+            buf += "timeout=%s\n" % (self.data.packages.timeout,)
```

Comment:
Why are you creating tuple from the `self.data.packages.timeout`? It should work with the plain variable.

Reply:
Fixed.